sandbox/package.json

```diff
@@ -4,30 +4,30 @@
   "private": true,
   "main": "index.js",
   "dependencies": {
-    "@expo/vector-icons": "^12.0.0",
+    "@expo/vector-icons": "^13.0.0",
     "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/bottom-tabs": "~6.0.9",
     "@react-navigation/native": "~6.0.6",
     "@react-navigation/stack": "~6.0.11",
-    "expo": "^44.0.0",
-    "expo-asset": "~8.4.4",
-    "expo-constants": "~13.0.0",
-    "expo-font": "~10.0.4",
-    "expo-linking": "~3.0.0",
-    "expo-splash-screen": "~0.14.0",
-    "expo-status-bar": "~1.2.0",
-    "expo-web-browser": "~10.1.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-native": "0.64.3",
-    "react-native-gesture-handler": "~2.1.0",
+    "expo": "^48.0.0",
```

Comment:
Can we split all these updates into a second PR, or do you think it's safe to do it all in one go?